### ISSUE TITLE
[Machine Learning] Redução de Dimensionalidade com PCA

### ISSUE THREAD

**Issue by davi-balsamao** [williamalmeidadev/deepfake-detection-research#11]
**Conceito**: A Análise de Componentes Principais (PCA) é uma técnica de redução de dimensionalidade que converte dados de alta dimensão em um novo conjunto de variáveis não correlacionadas (componentes principais)

**Por que estamos fazendo assim?** O aumento do número de features nos dados de deepfake leva à "maldição da dimensionalidade", que aumenta exponencialmente a complexidade e gera _overfitting_. O PCA captura a maior parte da variância com o menor número possível de colunas

**Problema que resolve**: Otimiza a performance computacional e limpa o excesso de ruído identificado previamente na Matriz de Correlação.

**Requisitos**: Aplicar a transformação utilizando a biblioteca `sklearn` sobre os dados já normalizados (StandardScaler).

**Código de Apoio:**
```py
from sklearn.decomposition import PCA
# Lógica: Não definimos o n_components estaticamente como "2" ou "3". Usamos float (ex: 0.95) para instruir o algoritmo a reter 95% da variância explicada, independentemente de quantos componentes isso exija.
pca = PCA(n_components=0.95)
df_pca = pca.fit_transform(df_cleaned_scaled)
print(f"Componentes retidos: {pca.n_components_}")
```

**Critérios de Aceite (Definition of Done):**
- Script Python executado aplicando o PCA nos dados numéricos .
- Gráfico gerado mostrando a Variância Explicada Cumulativa (Scree Plot).
- Dataset transformado exportado para o uso nos modelos subsequentes.

**Checklist**
- [ ] Importar PCA do módulo `sklearn.decomposition`.
- [ ] Calcular a matriz de covariância e encontrar autovalores/autovetores .
- [ ] Ajustar (`fit_transform`) o PCA no conjunto numérico.
- [ ] Documentar quantos componentes foram necessários para reter 95% da variância.
